src/pages/carbonMetrics/CarbonMetrics.tsx

```diff
@@ -0,0 +1,225 @@
+      return {
+        carbon: averages[averages.length - 1].carbon - averages[averages.length - 2].carbon,
+        percent: averages[averages.length - 1].percent - averages[averages.length - 2].percent
+      };
+    } else {
+      return {
+        carbon: 0,
+        percent: 0
+      };
+    }
+  };
+
+  return (
+    <Layout>
+      <Helmet>
+        <meta name="description" content="Boston Scientific Anatomy Design System website carbon metrics" />
+      </Helmet>
+      <div className="docs-body-minimal">
+        <main id="mainContent">
+          <div className="docs-page-header">
+            <div className="docs-metadata">
+              <h1 className="docs-title">Carbon metrics</h1>
+              {!!latestData && (
+                <dl className="docs-datestamp">
+                  <dt>Last Updated:</dt>
+                  <dd>{new Date(lastUpdated).toLocaleDateString()}</dd>
+                </dl>
+              )}
+            </div>
+          </div>
+          {!!isDataReady && allData.length > 0 && (
+            <>
+              {!!averages && (
+                <div className="docs-metrics-tile-grid">
+                  <div className="docs-metrics-tile">
+                    <h3>
+                      CO<sub>2</sub> emissions
+                    </h3>
+                    <p className="docs-metrics-tile-description">Average grams of carbon emitted per page.</p>
+                    <p className="docs-metrics-tile-value">{averages[averages.length - 1].carbon.toPrecision(3)}g</p>
+                    <p className="docs-metrics-tile-change">
+                      {getLatestChanges().carbon >= 0 && '+'}
+                      {getLatestChanges().carbon}g
+                    </p>
+                    <SparkChart data={averages} yValue="carbon" />
+                  </div>
+                  <div className="docs-metrics-tile">
+                    <h3>Percent cleaner</h3>
+                    <p className="docs-metrics-tile-description">
+                      Average score against other sites tested with Website Carbon.
+                    </p>
+                    <p className="docs-metrics-tile-value">{averages[averages.length - 1].percent.toFixed(2)}%</p>
+                    <p className="docs-metrics-tile-change">
+                      {getLatestChanges().percent >= 0 && '+'}
+                      {getLatestChanges().percent}%
+                    </p>
+                    <SparkChart data={averages} yValue="percent" />
+                  </div>
+                </div>
+              )}
+              <h3>Pages</h3>
+              <table className="docs-table-responsive">
+                <caption className="bsds-visually-hidden">
+                  Most recent carbon data for each page across the Anatomy documentation site.
+                </caption>
+                <thead>
+                  <tr>
+                    <th className="docs-table-header-sortable" aria-sort={getSortAria('url')}>
```

Comment:
can we check w3c patterns for sorting
https://www.w3.org/WAI/ARIA/apg/patterns/grid/

we can make changes in a future PR or ticket if needed